tutorials/abap-environment-business-service-provisioning/abap-environment-business-service-provisioning.md

```diff
@@ -15,7 +15,7 @@ author_profile: https://github.com/niloofar-naseri
   - How to create a service binding
   - How to create a communication scenario
 
-In this tutorial, wherever `XXX` appears, use a number (e.g. `000`).
+  In this tutorial, wherever `XXX` appears, use a number e.g. `000` or your initials.
```

Comment:
Please add comma before e.g.
Consider using "for example" instead of "e.g." for better readability.